src/renderer/index.html

```diff
@@ -9,43 +9,127 @@
+                <div class="sidebar-header">
+                    <h2 class="sidebar-title">Tools Marketplace</h2>
+                </div>
+                <div class="sidebar-body">
+                    <div class="marketplace-search">
+                        <input type="text" id="marketplace-search-input" class="search-input" placeholder="Search tools...">
+                    </div>
+                    <div class="marketplace-tools-list" id="marketplace-tools-list">
+                        <!-- Marketplace tools will be populated here -->
+                    </div>
+                </div>
+            </div>
+
+            <!-- Settings Sidebar -->
+            <div class="sidebar-content" id="sidebar-settings">
+                <div class="sidebar-header">
+                    <h2 class="sidebar-title">Settings</h2>
+                </div>
+                <div class="sidebar-body">
+                    <div class="settings-container-sidebar">
+                        <div class="setting-group">
+                            <label class="setting-label" for="sidebar-theme-select">Theme</label>
+                            <select id="sidebar-theme-select" class="setting-input">
+                                <option value="system">System</option>
+                                <option value="light">Light</option>
+                                <option value="dark">Dark</option>
+                            </select>
+                        </div>
+                        <div class="setting-group">
+                            <label class="setting-label" for="sidebar-auto-update-check">Auto Update</label>
+                            <input type="checkbox" id="sidebar-auto-update-check" class="setting-checkbox">
+                            <span class="setting-hint">Automatically check for updates</span>
+                        </div>
+                        <div class="setting-group">
+                            <button class="btn btn-primary" id="sidebar-save-settings-btn">Save Settings</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
         </aside>
 
         <!-- Main Content -->
         <main class="main-content">
             <!-- Tool Panel (for launched tools) -->
             <div id="tool-panel" class="tool-panel" style="display: none;">
                 <div class="tool-panel-header">
-                    <div class="tool-panel-title">
-                        <span class="tool-panel-icon">🔧</span>
-                        <span id="tool-panel-name">Tool Name</span>
+                    <div class="tool-tabs" id="tool-tabs">
+                        <!-- Tabs will be dynamically added here -->
+                    </div>
+                    <div class="tool-panel-controls">
+                        <select id="tab-connection-selector" class="connection-selector" title="Select connection for active tool">
```

Comment:
Dont need connection selector. When the tool is launched if the connection was already established it uses the same; if no connection was established then show a popup with list of connections to pick from for establishing the connection. This information is showcased in the footer. When the tab is changed the connection information in the footer changes.